src/plugins/colorchooser/structure_plugin.js

```diff
@@ -158,9 +161,9 @@ define( ['structure_custom', 'css!plugins/colorchooser/farbtastic/farbtastic.css
                 color.substring(7);
         break;
       case 'rgb':
-        wData.bus_r = value[0];
-        wData.bus_g = value[1];
-        wData.bus_b = value[2];
+        wData.bus_r = value[0] * 100 / 255.0;
+        wData.bus_g = value[1] * 100 / 255.0;
+        wData.bus_b = value[2] * 100 / 255.0;
         color = color.substring(0,1) +
```

Comment:
The same as above. Is that really correct?

By comparing the cases I'd expect that lines 168, 169 and 170 should have been changed to `toHex( value[...] * 255/100 )` instead